lib/membrane-element-rtp-aac/depayloader.ex

```diff
@@ -19,27 +19,27 @@ defmodule Membrane.RTP.AAC.Depayloader do
               ]
 
   @impl true
-  def handle_init(options) do
-    {:ok, Map.merge(options, %{leftover: <<>>})}
+  def handle_init(_ctx, options) do
+    {[], Map.merge(options, %{leftover: <<>>})}
```

Comment:
`Map.merge(options, ...)` will return `%{__struct__: __MODULE__, ...}`. Beyond this, use `Map.put/3` instead of `Map.merge/2` in this case

```suggestion
state = 
  options 
  |> Map.from_struct()
  |> Map.put(:leftover, <<>>)

{[], state}
```

Reply:
done